spec/type.dd

```diff
@@ -558,6 +558,38 @@ $(H3 $(LNAME2 string, $(D string)))
 
     $(P A $(DDSUBLINK spec/arrays, strings, $(I string) is a special case of an array.))
 
+$(H3 $(LNAME2 noreturn, $(D noreturn)))
+
+    $(P `noreturn` is the $(LINK2 https://en.wikipedia.org/wiki/Bottom_type, bottom type)
+    which can implicitly convert to any type, including `void`.
+    A value of type `noreturn` will never be produced and the compiler can
+    optimize such code accordingly.)
+
+    $(P A function that never returns has the return type `noreturn`. This can
+    occur due to an infinite loop or always throwing an exception.
+    This is particularly useful when inferring types.)
```

Comment:
I don't think sentence adds value, it's unspecific.

Reply:
Removed now.